mainwindow.cc

```diff
@@ -3105,7 +3105,13 @@ void MainWindow::hotKeyActivated( int hk )
     toggleMainWindow();
   else
   if ( scanPopup.get() )
+#ifdef HAVE_X11
+    // call scanPopup directly may cause clipboard conflicts with other Qt programs
+    // on Linux and make popup failed, wait a little while to fix it.
```

Comment:
> Just tested the fix briefly on my system where Ctrl+C+C has worked perfectly in all applications. Haven't noticed any regressions. I think the 10 ms delay is small enough to be pretty safe regression-wise.

The reason I overthought this issue before was because I was afraid that the time would vary dramatically in different computers, especially old computers without SDD. But it looks like 10ms is basically enough. As my oldest computer is only 5 years old, so I cannot say that all systems are applicable. Anyway, it should solve most problems.

Reply:
> for some reason, you silently ignore some of my suggestions in this and previous PR.
> 
> So again: I suggest referencing the fixed issues in the commit message body, not only in the PR description. That way the context would be easier to find for future developers. They won't have to look for the related PR first, but could navigate to the issues directly from git log.

Sorry for that, I hope I finally did it right this time. It's hard to do something not familiar with, but I will try my best. Thank you again.